content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -2,135 +2,131 @@
-options, then click :guilabel:`Save` to save all changes.
-
-Pricelists
-==========
-
 After activating and saving the :guilabel:`Pricelists` feature, the :guilabel:`Settings` page
-reloads and, from here, either select the :guilabel:`Pricelists` link (beneath the
+reloads. From here, either select the :guilabel:`Pricelists` link (beneath the
 :guilabel:`Pricelists` feature on the :guilabel:`Settings` page), or navigate to
 :menuselection:`Sales app --> Products --> Pricelists`.
 
 Either option reveals the :guilabel:`Pricelists` page, in which pricelists can be created and/or
 modified at any time.
 
-.. image:: pricing/pricelists-page.png
-   :align: center
+.. image:: pricing/18-sales-pricelist-page.png
    :alt: How the pricelists page looks in Odoo Sales.
 
 .. important::
-   The order of the pricelists on the :guilabel:`Pricelists` page has an impact on how they are
-   applied. If several pricelists share the same criteria, **only** the first listed pricelist is
-   applied.
-
-   For example, for two pricelists with different rules, but same criteria (e.g., same website, same
-   country), only the *first* pricelist in the list is applied.
+   If there is no specific pricelist configured on a sales quotation, the :guilabel:`Default`
+   pricelist is applied.
 
 .. note::
-   The :guilabel:`Public Pricelist` is the default pricelist used with Odoo *Sales* and *eCommerce*.
-   This pricelist is applied by default, if there are no matching criteria.
+   The :guilabel:`Selectable` column is only applicable to Odoo **eCommerce**. This option allows
+   website visitors to choose a pricelist when shopping in your **eCommerce** website.
 
 .. note::
-   In Odoo 17 (and above), it is no longer required to have a pricelist entered in the
-   :guilabel:`Pricelist` field on a quotation form in order to confirm it (i.e. turn it into a sales
-   order).
+   In Odoo 17 (and above), you are *not required* to enter a pricelist in the :guilabel:`Pricelist`
+   field on a sales quotation form in order to confirm it (i.e. turn it into a sales order).
+
+   The chatter section is available on pricelist forms, which allows you to add
+   notes and communications on each pricelist page.
 
-   It should also be noted that, in Odoo 17 (and above), a chatter section can be found on pricelist
-   forms, which enhances the ability to communicate about them.
+Creating and editing pricelists
+-------------------------------
 
-From the :guilabel:`Pricelists` page, either select the desired pricelist to edit, or click
+From the :guilabel:`Pricelists` page, either select the pricelist to edit, or click
 :guilabel:`New` to create a new pricelist, which reveals a blank pricelist form that can be
 configured in a number of different ways.
 
-.. image:: pricing/pricelist-detail-form.png
-   :align: center
+.. image:: pricing/18-sales-new-pricelist-form.png
    :alt: How the pricelist detail form looks in Odoo Sales.
```

Comment:
This image (`18-sales-new-pricelist-form.png`) is too wide at 1231 px and is uncompressed, please reduce its width to be at or below 933px and please compress with pngquant